gpuarray/test/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ if(NOT CHECK_FOUND)
 endif()
 
 if(CHECK_FOUND)
-enable_testing()
```

Comment:
Hm for some reason cmake does not create a "test" make target when `enable_testing()` is included in a subdirectory's CMakeLists.txt.